pkg/scale/uint128.go

```diff
@@ -155,7 +155,7 @@ func (u *Uint128) UnmarshalJSON(data []byte) error {
 
 	dec, err := NewUint128(intVal)
 	if err != nil {
-		return err
+		return fmt.Errorf("failed to unmarshal Uint128: %w:", err)
```

Comment:
- use `verb-ing something: %w` to wrap errors, since wrapping with `cannot verb something` or `failed to  verb something` adds a lot of `cannot`/`failed` repetition for deeply wrapped errors. Only the bottom wrapped error should state it failed at what and might use `cannot`/`failed to`.
- removed typo suffix `:`
- changed message to be more precise at what failed, since the caller to `(u *Uint128) UnmarshalJSON` would likely wrap the error with context `unmarshaling uint128` already
- lowercase `Uint128` since the `revive` linter complains 🗣️ 

```suggestion
		return fmt.Errorf("creating uint128 from big integer: %w", err)
```